docs/src/main/asciidoc/quartz.adoc

```diff
@@ -201,6 +201,7 @@ Edit the `application.properties` file and add the below configuration:
 # Quartz configuration
 quarkus.quartz.clustered=true <1>
 quarkus.quartz.store-type=jdbc-cmt <2>
+quarkus.quartz.misfire-policy.task-job=ignore-misfire-policy <3>
```

Comment:
The `task-job` is a job identity, or?

Reply:
Yes it is the job identity. 
@rmanibus whenever you get the time, would you mind mentioning that in the doc as well.